python/doc/pyplots/GeneralizedExtremeValue.py

```diff
@@ -15,7 +15,6 @@
     pdf_curve.setLegend(legend)
     cdf_curve.setLegend(legend)
     pdf_graph.add(pdf_curve)
-    cdf_graph.add(cdf_curve)
```

Comment:
why this deletion ?

Reply:
an error. I removed also the lines that were specifying the color of each Drawable.